components/dropdown/Dropdown.module.css

```diff
@@ -74,15 +76,18 @@
 }
 
 @media (min-width: 768px) {
-  .cancelBtn {
-    top: 115px;
-    left: 369px;
```

Comment:
Since we positioned relative, we don't need to adjust the position of the X button when the screen gets larger 🎉